### PR TITLE
feat/add chain values converters

### PR DESCRIPTION
**Pull request by aarmoa**
- Added new functions in all Market classes to convert values from extended chain format (the ones provided by chain streams) into human-readable format
- Updated proto definitions for Indexer node using version v1.12.79.1
- Updated market and tokens INI configuration files

### CODE REVIEW

File: client/core/market.go [hunk 1]
```diff
@@ -46,6 +46,18 @@ func (spotMarket SpotMarket) PriceFromChainFormat(chainValue cosmtypes.Dec) deci
...
+func (spotMarket SpotMarket) QuantityFromExtendedChainFormat(chainValue cosmtypes.Dec) decimal.Decimal {
+	return spotMarket.fromExtendedChainformat(spotMarket.QuantityFromChainFormat(chainValue))
+}
+
+func (spotMarket SpotMarket) PriceFromExtendedChainFormat(chainValue cosmtypes.Dec) decimal.Decimal {
+	return spotMarket.fromExtendedChainformat(spotMarket.PriceFromChainFormat(chainValue))
+}
+
+func (spotMarket SpotMarket) fromExtendedChainformat(chainValue decimal.Decimal) decimal.Decimal {
+	return chainValue.Div(decimal.New(1, AdditionalChainFormatDecimals))
```
Comment (vinhphuctadang) [inline client/core/market.go:58]: Better to do this in utils file, this one doesn't depend on `spotMarket` or `derivativeMarket`

```
fromExtendedChainformat(chainValue decimal.Decimal) decimal.Decimal
```

But why chainValue has decimal.Decimal type?

Comment (aarmoa) [reply client/core/market.go:58]: Ok, will move it (unfortunately moving it to utils makes it less OOP, something that I don't really like)

Regarding the chainValue, it was not originally decimal.Decimal. But the function that calls this one first translates the value according to market tokens decimals, making it a decimal.Decimal

Comment (vinhphuctadang) [reply client/core/market.go:58]: > unfortunately moving it to utils makes it less OOP, something that I don't really like

Yeah it's a tool and we need flexibility to choose which to use

> Regarding the chainValue, it was not originally decimal.Decimal. But the function that calls this one first translates the value according to market tokens decimals, making it a decimal.Decimal

Oh I thought using sdk.Dec is enough when chain streamed the value

**Review by vinhphuctadang — APPROVED** [review]
please fix lint issue

**Pull request merged**